jobbergate-api/jobbergate_api/main.py

```diff
@@ -85,3 +87,14 @@ async def disconnect_database():
     """
     logger.debug("Disconnecting database")
     await database.disconnect()
+
+
+@app.exception_handler(RequestValidationError)
+async def validation_exception_handler(request: Request, err: RequestValidationError):
+    """
+    Handle exceptions from pydantic validators.
+    """
+    raise HTTPException(
+        status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+        detail=f"Validation error for request to {request.url} with data: {request.json()}: {err}",
+    )
```

Comment:
won't FasAPI itself do something very similar to this?

Reply:
Well, you will get a 422 and a response that includes data about what is wrong with the request. However, you won't see it in the logs at all. It will just show something like "Request Failed: 422" or something like that with no details. It makes it very frustrating to debug because you have to re-issue the same request and look at the response.